web/libs/js/main.dash2.js

```diff
@@ -1508,7 +1508,7 @@ $.ccio.globalWebsocket=function(d,user){
                                 if(d.d.stream_flv_maxLatency&&d.d.stream_flv_maxLatency!==''){
```

Comment:
Just randomly saw this and I really have to ask... what's with the lack of spaces?

Reply:
what? you mean like before and after `&&` ?